content/en/platform/corda/5.2/deploying-operating/keys/_index.md

```diff
@@ -0,0 +1,196 @@
+    {\"alias\":\"master2\",\"passphrase\":{\"configSecret\": {\"encryptedSecret\":\"gzXRjE4AoZSW8GQhcqbw0shQfwWGy5O8alNDqXaqTUzyUnLBVUXLwhv+uAEZzecQixiaEaNtfOvuftYW9NjwwRyf6m4KZIag\"}},\"salt\": \
+    {\"configSecret\":{\"encryptedSecret\":\"flvAnf5G/aRmDIaahXHjftEpc0odufhzZsuiipxtQTFV+pccB+RHnIqle26jjDlDUBfn3HPbVezKhpRHL/NP5M+cX/rU15DE\"}}}]}, \
+    \"retrying\":{\"maxAttempts\": {\"default\": 3},\"waitBetweenMills\":{\"default\":[200]}}}}","schemaVersion": {"major":1, "minor":0}}' \
+    "$REST_API_URL/config"
+   ```
+   {{% /tab %}}
+   {{% tab name="PowerShell" %}}
+
+   ```shell
+  Invoke-RestMethod -SkipCertificateCheck -Headers @{Authorization=("Basic {0}" -f ${REST_API_USER}:${REST_API_PASSWORD})} -Method Put -Uri "$REST_API_URL/config" -Body (ConvertTo-Json -Depth 4 @{
+   {
+     "config": "{
+        "caching":{
+          "expireAfterAccessMins":{
+            "default":60
+          },
+          "maximumSize":{
+            "default":10000
+          }
+        },
+        "hsm":{
+          "defaultWrappingKey": "master1",
+          "retrying":{
+            "attemptTimeoutMills": 20000,
+            "maxAttempts": 3
+          },
+          "wrappingKeys":[{
+            "alias": "master1",
+            "passphrase":{
+              "configSecret":
+              {
+                "encryptedSecret": "gzXRjE4AoZSW8GQhcqbw0shQfwWGy5O8alNDqXaqTUzyUnLBVUXLwhv+uAEZzecQixiaEaNtfOvuftYW9NjwwRyf6m4KZIag"
+              }
+            },
+            "salt":{
+              "configSecret":
+              {
+                "encryptedSecret": "cvNVJ4D7fNGm59Uh3Tlhi0W3pToDxl7vykUdZaxEuEPGUMdmu7G+xTEfl0CcC2R9aWl6FgrzmX5x+qCGNElUi/PBA40EGjEU"
+              }
+            },
+          },
+          {
+            "alias": "master2",
+            "passphrase":{
+              "configSecret":
+              {
+                "encryptedSecret": "ZAmGcppwqUNsViSE06b801DVARbD9wT7SZnYP03WKZFOnC9KTsII04pWWq5Je2CUZdP+xIsHFUO3RkZVwhwnyQhWoxcW6JEo"
+              }
+            },
+            "salt":{
+              "configSecret":
+              {
+                "encryptedSecret": "flvAnf5G/aRmDIaahXHjftEpc0odufhzZsuiipxtQTFV+pccB+RHnIqle26jjDlDUBfn3HPbVezKhpRHL/NP5M+cX/rU15DE"
+              }
+            }
+          }]
+        "retrying":{
+          "maxAttempts":{
+            "default":3
+          },
+          "waitBetweenMills":{
+            "default":[200]
+          }
+        }
+      }",
+      "schemaVersion": {
+        "major": 1,
+        "minor": 0
+      },
+      "section": "corda.crypto",
+      "version": 1
+   })
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+3. Rotate the key to the new default key using the POST method of the [/api/v5_2/wrappingkey/rotation/{tenantid} endpoint](../../reference/rest-api/openapi.html#tag/Key-Rotation-API/operation/post_wrappingkey_rotation__tenantid_):
```

Comment:
Rotate the old master key to the new default master key. After this step all managed wrapping keys will be encrypted with the new master key.